### PR TITLE
Adding direct links to the files from the README.

### PR DESCRIPTION
**Pull request by Sfshaza**

### CODE REVIEW

File: test/example/iss/README.md [hunk 1]
```diff
@@ -40,19 +43,24 @@ between them.
       Point<double> london = new Point(51.5073, -0.1277);
       Point<double> paris = new Point(48.8566, 2.3522);
       double d = sphericalDistanceKm(london, paris);
+      // London should be approximatley 343.5km
```
Comment (kwalrath) [inline test/example/iss/README.md]: typo in "approximately"

File: test/example/iss/README.md [hunk 2]
```diff
@@ -40,19 +43,24 @@ between them.
...
+      // (+/- 0.1km) from Paris.
       expect(d, closeTo(343.5, 0.1));
     });
 
     test('San Francisco - Mountain View', () {
       Point<double> sf = new Point(37.783333, -122.416667);
       Point<double> mtv = new Point(37.389444, -122.081944);
       double d = sphericalDistanceKm(sf, mtv);
+      // San Francisco should be approximatley 52.8km
```
Comment (kwalrath) [inline test/example/iss/README.md]: typo in "approximately"

File: test/example/iss/README.md [hunk 3]
```diff
@@ -12,9 +12,11 @@ expected outcome.
 The ISS library, `iss.dart`, consists of two classes:
 
 **IssLocator**
+
```
Comment (kwalrath) [inline test/example/iss/README.md]: Why add these newlines? They make the `<dl>` code (implemented with the `:` on the following line) break.

Comment (Sfshaza) [inline test/example/iss/README.md]: D'OH. Yes, I wasn't thinking...

**Pull request merged**